libs/langchain/tests/unit_tests/test_text_splitter.py

```diff
@@ -240,32 +235,31 @@ def __test_iterative_text_splitter(chunk_size: int, keep_separator: bool) -> Lis
 def test_iterative_text_splitter() -> None:
     """Test iterative text splitter."""
     text = """Hi.\n\nI'm Harrison.\n\nHow? Are? You?\nOkay then f f f f.
-This is a weird text to write, but gotta test the splittingggg some how.
+This is a weird to write, but gotta test the splittingggg some how.
 
 Bye!\n\n-H."""
-    splitter = RecursiveCharacterTextSplitter(chunk_size=10, chunk_overlap=1)
+    chunk_size = 10
+    splitter = RecursiveCharacterTextSplitter(chunk_size=chunk_size, chunk_overlap=1)
     output = splitter.split_text(text)
     expected_output = [
-        "Hi.",
-        "I'm",
+        "Hi.\n\nI'm H",
         "Harrison.",
-        "How? Are?",
-        "You?",
+        ".\n\nHow? Ar",
```

Comment:
hm seems like the original behavior was nicer?

Reply:
Good point. Fixed that.